src/models/timeaxisheaderview.cpp

```diff
@@ -0,0 +1,94 @@
+#include "timeaxisheaderview.h"
+
+#include <QDebug>
+#include <QPainter>
+#include <QtMath>
+
+#include "eventmodel.h"
+#include "filterandzoomstack.h"
+#include "util.h"
+
+#include "../../3rdparty/PrefixTickLabels/src/PrefixTickLabels.h"
+
+TimeAxisHeaderView::TimeAxisHeaderView(FilterAndZoomStack* filterAndZoomStack, Qt::Orientation orientation,
+                                       QWidget* parent)
+    : QHeaderView(orientation, parent)
+    , m_filterAndZoomStack(filterAndZoomStack)
+{
+    setMinimumHeight(37);
```

Comment:
this hardcoded number won't work, it's probably dependent on the font height? so something like using the double line height from the fontmetrics?

Reply:
Changed to be calculated based on the font size.